lgsm/functions/command_postdetails.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+# LGSM command_postdetails.sh function
+# Author: CedarLUG
+# Contributor: CedarLUG
+# Website: https://gameservermanagers.com
+# Description: Strips sensitive information out of Details output
+
+local commandname="POSTDETAILS"
+local commandaction="Postdetails"
+local function_selfname="$(basename $(readlink -f "${BASH_SOURCE[0]}"))"
+
+# POSTDETAILS variable affects the output of command_details.sh.  Setting
+# it here silences the output from sourcing command_details.sh.
+POSTDETAILS=yes
+
+# Set POSTTARGET to the appropriately-defined post destination. 
+ 
+# The options for POSTTARGET are:
+# The default destination - hastebin 
+# POSTTARGET="http://hastebin.com"
+#
+# Secondary destination - pastebin
+# POSTTARGET="http://hastebin.com
+#
+# Third option - leave on the filesystem
+# POSTTARGET=
+#
+# All of these options can be specified/overridden from the top-level
+# invocation, as in:
+#  rustserver@gamerig:~$ POSTTARGET="http://pastebin.com" ./rustserver pd
+# to post to pastebin, or
+#  rustserver@gamerig:~$ POSTTARGET= ./rustserver pd
+# to leave the output on the filesystem.
+POSTTARGET=${POSTTARGET="http://hastebin.com"}
+
+# For pastebin, you can set the expiration period.
+POSTEXPIRE="1W" # use 1 week as the default, other options are '24h' for a day, etc.
+
+# This file sources the command_details.sh file to leverage all
+# of the already-defined functions.  To keep the command_details.sh
+# from actually producing output, the main executable statements have
+# been wrapped in the equivalent of an ifdef clause, that looks
+# for the variable "postdetails" to be defined. -CedarLUG
+
+# source all of the functions defined in the details command
+. ${functionsdir}/command_details.sh
```

Comment:
all dirs should have quotes around them to prevent issues with dirs that have spaces ```"${functionsdir}"/command_details.sh```